fili-sql/src/test/groovy/com/yahoo/bard/webservice/sql/builders/SimpleDruidQueryBuilder.groovy

```diff
@@ -0,0 +1,229 @@
+                        COUNTRY_ISO_CODE, IS_NEW, IS_ROBOT, PAGE,
+                        USER, COMMENT, IS_UNPATROLLED, NAMESPACE,
+                        COUNTRY_NAME, CITY_NAME, IS_MINOR, IS_ANONYMOUS,
+                        REGION_ISO_CODE, CHANNEL, REGION_NAME, METRO_CODE
+                ),
+                apiPrepend,
+                fieldPrepend
+        )
+    }
+
+    public static TimeSeriesQuery timeSeriesQuery(
+            String name,
+            Filter filter,
+            Granularity timeGrain,
+            List<String> metrics,
+            List<String> dimensions,
+            List<Aggregation> aggregations,
+            List<PostAggregation> postAggs,
+            List<Interval> intervals
+    ) {
+        return new TimeSeriesQuery(
+                dataSource(name, metrics, dimensions),
+                timeGrain,
+                filter,
+                aggregations,
+                postAggs,
+                intervals
+        );
+    }
+
+    public static GroupByQuery groupByQuery(
+            String name,
+            Filter filter,
+            Having having,
+            List<Dimension> groupByDimensions,
+            Granularity timeGrain,
+            List<String> metrics,
+            List<String> dimensions,
+            List<Aggregation> aggregations,
+            List<PostAggregation> postAggs,
+            List<Interval> intervals,
+            LimitSpec limitSpec
+    ) {
+        return new GroupByQuery(
+                dataSource(name, metrics, dimensions),
+                timeGrain,
+                groupByDimensions,
+                filter,
+                having,
+                aggregations,
+                postAggs,
+                intervals,
+                limitSpec
+        )
+    }
+
+    public static TableDataSource dataSource(String name, List<String> metrics, List<String> dimensions) {
+        return dataSource(name, DefaultTimeGrain.DAY, DateTimeZone.UTC, metrics, dimensions, "", "")
+    }
+
+
+    public static TableDataSource dataSource(
+            String name,
+            ZonelessTimeGrain zonelessTimeGrain,
+            DateTimeZone dateTimeZone,
+            List<String> metrics,
+            List<String> dimensions,
+            String apiPrepend,
+            String fieldPrepend
+    ) {
+        ZonedTimeGrain zonedTimeGrain = new ZonedTimeGrain(zonelessTimeGrain, dateTimeZone);
+        Set<Column> columns = setOf();
+        Map<String, String> logicalToPhysicalColumnNames = new HashMap<>()
+        metrics.forEach { logicalToPhysicalColumnNames.put(apiPrepend + it, fieldPrepend + it) }
+        dimensions.forEach { logicalToPhysicalColumnNames.put(apiPrepend + it, fieldPrepend + it) }
+
+        DataSourceMetadataService metadataService = new DataSourceMetadataService();
+        metadataService.update(
+                DataSourceName.of(name),
+                new DataSourceMetadata(name, Collections.emptyMap(), Collections.emptyList())
```

Comment:
It's Groovier to use `[:]` and `[]` rather than `Collections.emptyMap` and `Collections.emptyList`.